src/vaadin-message-list.js

```diff
@@ -69,16 +69,25 @@ class MessageListElement extends ElementMixin(ThemableMixin(PolymerElement)) {
         :host([hidden]) {
           display: none !important;
         }
+
+        ol {
+          list-style: none;
+          margin: 0;
+        }
       </style>
-      <template is="dom-repeat" items="[[items]]">
-        <vaadin-message time="[[item.time]]" user="[[item.user]]">[[item.text]]</vaadin-message>
-      </template>
+      <ol>
+        <template is="dom-repeat" items="[[items]]">
+          <vaadin-message time="[[item.time]]" user="[[item.user]]">[[item.text]]</vaadin-message>
```

Comment:
I think you have to add `<li>` inside the `<ol>` for valid html.

Reply:
I'm not really a fan of complicating our DOM structure with `<ol>` and `<li>`. I saw the comment that you should avoid role="" if you can do it with the correct native HTML elements, but I still think it brings in more possible issues than what benefits we get for it. Like deeper DOM, and possible unforeseen css rules in different environments. Other components in Vaadin has already used `role="list"` and `role="listitem"` and we will ensure accessibility compatibility.

Reply:
@Peppe understandable! I would remove `<ol>` then and use `<div role=list>` or something else. Looking at the HTML spec using `<ol>` without `<li>` is a violation: https://www.w3.org/TR/html52/grouping-content.html#the-ol-element

Reply:
Fixed!

Reply:
Updated my #14 - looks good!